kratos/python/add_matrix_to_python.cpp

```diff
@@ -76,6 +76,22 @@ namespace Python
         //here we add the dense matrix
         auto matrix_binder = CreateMatrixInterface< DenseMatrix<double> >(m,"Matrix");
         matrix_binder.def(py::init<const DenseMatrix<double>::size_type, const DenseMatrix<double>::size_type>());
+        matrix_binder.def("__init__", [](DenseMatrix<double> &m, py::buffer b){
```

Comment:
@Rbravo555 you are the one that has this clearer.

it looks good on my side though

Reply:
It does look good to me as well. Thanks!

Reply:
an you use a different variable name here?
In most cases `m` is the pybind-module